scripts/import/map_hgmd_coord.pl

```diff
@@ -67,19 +67,18 @@ =head1 CONTACT
 $TMP_FILE = $ImportUtils::TMP_FILE;
 
 Bio::EnsEMBL::Registry->load_all( $registry_file );
-my $cdb2 = Bio::EnsEMBL::Registry->get_DBAdaptor($species,'core');
-my $vdb2 = Bio::EnsEMBL::Registry->get_DBAdaptor($species,'variation');
-my $dbCore2 = $cdb2->dbc->db_handle;
-my $dbVar2 = $vdb2->dbc->db_handle;
+my $cdb = Bio::EnsEMBL::Registry->get_DBAdaptor($species,'core');
+my $vdb = Bio::EnsEMBL::Registry->get_DBAdaptor($species,'variation');
+my $dbVar = $vdb->dbc->db_handle;
 
-my $sa2 = $cdb2->get_SliceAdaptor();
+my $sa2 = $cdb->get_SliceAdaptor();
```

Comment:
The 2 has been removed from other variables. For consistency suggest dropping for $sa2 as well.